demo/README.md

```diff
@@ -0,0 +1,56 @@
+# Demo
+
+This demo showcases a TLS-protected exchange that uses attestation evidence instead of X.509 credentials to identify and authenticate a client to a server.
+
+In [RATS terms](https://www.rfc-editor.org/rfc/rfc9334.html#figure-1), the TLS client is the attester, while the TLS server is the relying party.
+
+The relying party delegates checking the trustworthiness of the evidence presented by the attester to an attestation verifier, using a [background check](https://www.rfc-editor.org/rfc/rfc9334.html#figure-6) arrangement.
+
+A successful handshake is predicated on the provisioning of the reference values and endorsements associated with the attester to the verifier.
+
+```mermaid
+flowchart TB
+    subgraph Attester
+    end
+
+    subgraph RP ["Relying Party"]
+    end
+
+    subgraph Verifier
+    end
+
+    subgraph RVP[Reference Value & </br>Endorsement Provider]
+    end
+
+    RP <-- Attested TLS --> Attester
+    RP -- "Veraison</br>Verification API" --> Verifier
+    RVP -- "Veraison</br>Provisioning API" --> Verifier
+```
+
+## Actions
+
+**Note**: All the `make(1)` targets below must be invoked in the current directory.
+
+1. Start up the demo environment:
+
+```shell
+make start
+```
+
+2. Obtain verification keys and PCR golden values from the TPM, repackage into [CoRIM](https://datatracker.ietf.org/doc/draft-ietf-rats-corim), and send to the verifier:
```

Comment:
```suggestion
2. Obtain verification keys and PCR golden values from the TPM, repackage into [CoRIM](https://datatracker.ietf.org/doc/draft-ietf-rats-corim), and send to the Verifier:
```

Reply:
I don't think this is necessary, there is no ambiguity in this context I think.

If we wanted to do the "RATS capitalisation" we should apply it to all the terms of art (evidence, relying party, attester, endorsements, reference values...)

Reply:
Yes, we should as we are referring to RATS specifically on the README + the pictures show that way

Reply:
as I said, I don't think it is necessary.  There is zero risk someone is getting confused by the absence of a capital letter.

Reply:
its not the question of necessity, but consistency.
It is how they will be documented in RATS Arch as well as in CoRIM.

Reply:
ok, since this is not necessary I reckon we can merge it as-is and raise an issue for an editorial pass that aligns this README with RFC9334

Reply:
I can make the edits myself

Reply:
@thomas-fossati  Have done it now, please check:

Reply:
cool, thanks